changelog.sh

```diff
@@ -85,6 +85,11 @@ if [ ! -z "$INPUT_GITHUB_API" ]; then
   ARG_GITHUB_API="--github-api $INPUT_GITHUB_API"
 fi
 
+ARG_EXCLUDE_TAGS_REGEX=""
+if [ ! -z "$ARG_EXCLUDE_TAGS_REGEX" ]; then
```

Comment:
if and assigned value should be `$INPUT_EXCLUDE_TAGS_REGEX`

Reply:
Fixed!